### ISSUE TITLE
Warning from future non-tuple

### ISSUE THREAD

**Issue by londumas** [desihub/speclite#44]
I get this warning message, running quickquasar.
It is only a warning for now, but it might be useful to fix it before it becomes an error.

```
quickquasars -i /project/projectdirs/desi/mocks/lya_forest/london/v6.0/v6.0.0/1/137/transmission-16-137.fits
--outdir /project/projectdirs/desi/mocks/lya_forest/london/v6.0/v6.0.0/eboss-0.0/test-0-0/spectra-16
--zbest --bbflux --eboss --seed 123
--shift_kms_los -100.0 --gamma_kms_zfit 400.0 --zmin 1.8 --metals LYB
--overwrite
```
```
<home>/Programs/dkirkby/speclite/speclite/filters.py:1466: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  values_no_units = values_no_units[values_slice]
<home>/Programs/dkirkby/speclite/speclite/filters.py:1520: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  integrand = integrand[values_slice]

```

**Comment by londumas** [comment]
@dkirkby, I can not push a branch to you pakage to correct this, can we take care of that?
```remote: Permission to dkirkby/speclite.git denied to londumas.
fatal: unable to access 'https://github.com/dkirkby/speclite.git/': The requested URL returned error: 403
``

**Comment by dkirkby** [comment]
Thanks for following up on this. I just added you as a collaborator so you can make a PR.

**Comment by weaverba137** [comment]
This was closed by #52.
